### PR TITLE
Created Net Device Reboot Button FAQ page

### PR DESCRIPTION
**Pull request by trevorbecker**
Created Net Device Reboot Button FAQ page

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-d29fb56254/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-9774325aae/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-ef93baed9c/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-285a221a7d/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-4f153857ab/how-to/)

**Comment by kmsholcomb** [comment]
Hi, @DonNateR. My suggested edits are in the attached Word doc (and PDF copy). Most of the edits are pretty straightforward. My main question is around the interchangeable use of _appliance_ and _device_. See my comments in the file (probably easiest to see them in the PDF). Thanks!

[Net Device Reboot Button FAQ_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/709353/Net.Device.Reboot.Button.FAQ_editsKH.pdf)
[Net Device Reboot Button FAQ_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/709354/Net.Device.Reboot.Button.FAQ_editsKH.docx)

**Comment by DonNateR** [comment]
@trevorbecker: Can you clarify the difference between the terms "device" and "appliance"? Right now they are used fairly interchangeably in the article.

**Comment by trevorbecker** [comment]
@DonNateR @kmsholcomb I think you bring up a great point on consistency. Since there is no difference, all references of network appliance can be replaced with network device.

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-a34bb6d182/how-to/)

**Comment by DonNateR** [comment]
@trevorbecker: Can't get collapsable lists to work without changing a few things on the back end. Are you ok with this preview format? https://staging.developer.rackspace.com/support.rackspace.com/build-a34bb6d182/how-to/net-device-reboot-faq/

**Comment by trevorbecker** [comment]
@DonNateR Yes, we are okay with this
thank you

File: content/dedicated-hosting/Net-Device-Reboot-Button-FAQ.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+The warm-boot process reboots your network appliance which resets all memory utilization. The device loads the preconfigured operating system from primary storage and reloads the device’s saved configuration.
+
+#### Why would I want to reboot my network device?
+
+Generally, you need to reboot a network device only when a new version of the operating system needs to be loaded or the operating system contains a bug that has caused processes on the device to operate improperly. Rackspace prestages the new operating system or the operating system updates on the network device. During the next reboot, the operating system is updated to the staged code. Rackspace notifies you when new code is staged on your device so that you are aware of any pending updates.
+ 
+#### When should I reboot my network devices?
+
+You can reboot your network deviceses whenever you like. Rackspace highly recommends that except in the case of emergency, you always reboot your network devices in a maintenance window when you expect the least impact to your production or business.
```
Comment (stephamon) [inline content/dedicated-hosting/Net-Device-Reboot-Button-FAQ.md]: "deviceses" should probably be "devices" :)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-4f225b2b42/how-to/).

**Comment by stephamon** [comment]
👍

**Pull request merged**